src/autolabel/tasks/question_answering.py

```diff
@@ -106,85 +100,24 @@ def get_explanation_prompt(self, example: Dict) -> str:
         )
 
     def eval(
-        self, llm_labels: List[LLMAnnotation], gt_labels: List[str]
+        self,
+        llm_labels: List[LLMAnnotation],
+        gt_labels: List[str],
+        additional_metrics: Optional[List[BaseMetric]] = [],
```

Comment:
lmk if passing additional metrics over a set of default metrics for a task makes sense - we could have this be a set of metrics so you can also remove some default metrics

Reply:
would a user specify additional metrics in the config ? (or, in the future world, initialize it in code themselves and pass it here) ?

Reply:
The hope is that these Metrics will be defined in code and the user can specify them in the agent.run command

Reply:
Added a test to show how a new metric can be added